examples/tutorial/05_Interactive_Pipelines.ipynb

```diff
@@ -311,6 +311,54 @@
     "For readability, seven columns were chosen before displaying the `DataFrame`. Have a look at `df.columns` and pick a different set of columns for display."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Functions as inputs\n",
+    "\n",
+    "Quite often the data structure you want to explore in a pipeline, e.g. a Pandas DataFrame, may itself be the outcome of another pipeline. It may for instance be extracted and transformed from a database or an API call, or the output of some simulation or pre-processing. With `hvplot.bind` you can bind an input function that returns the data structure you want to explore, to widgets that when updated will re-execute that function. \n",
+    "\n",
+    "For the sake of simplicity we will not fetch data from an external location, instead we create a simple function that filters the earthquakes dataset by event type and returns a DataFrame."
```

Comment:
```suggestion
    "To keep this example self-contained we'll illustrate this process using a simple example that filters the earthquakes dataset by event type and returns a DataFrame, but this function could be _any_ computation that returns a DataFrame."
```

(but it seems like a more canonical example would be for a user to select one of several CSV files, which _requires_ an approach like this and is a very common need.)